packages/snaps-controllers/wdio.config.ts

```diff
@@ -0,0 +1,73 @@
+import { NodeGlobalsPolyfillPlugin } from '@esbuild-plugins/node-globals-polyfill';
+import { NodeModulesPolyfillPlugin } from '@esbuild-plugins/node-modules-polyfill';
+import type { Options } from '@wdio/types';
+import { resolve } from 'path';
+import tsconfigPaths from 'vite-tsconfig-paths';
+
+export const config: Options.Testrunner = {
+  runner: [
+    'browser',
+    {
+      headless: true,
+      viteConfig: {
+        plugins: [tsconfigPaths()],
+        optimizeDeps: {
+          esbuildOptions: {
+            plugins: [
+              NodeModulesPolyfillPlugin(),
+              NodeGlobalsPolyfillPlugin({
+                buffer: true,
+                process: true,
+              }),
+            ],
+          },
+        },
+      },
+    },
+  ],
+
+  autoCompileOpts: {
+    tsNodeOpts: {
+      project: './tsconfig.json',
+    },
+  },
+
+  specs: ['./src/**/*.test.browser.ts'],
+
+  maxInstances: 10,
+  capabilities: [
+    {
+      maxInstances: 5,
+      browserName: 'chrome',
+      acceptInsecureCerts: true,
+    },
+  ],
+
+  logLevel: 'silent',
+
+  services: [
+    'chromedriver',
+    [
+      'static-server',
+      {
+        port: 4567,
+        folders: [
+          {
+            mount: '/',
+            path: resolve(
+              __dirname,
+              '../snaps-execution-environments/__test__/iframe-test',
```

Comment:
Do we still need this to be an insecure build when using an actual browser?

Reply:
No, I ran tests using the secure build, but that one isn't built in CI and to avoid further build time I didn't add that. If you prefer I can change that though.

Reply:
I mean, it would be great if we could test it fully secured so we don't have to have the `iframe-test` build anymore. We can do it iteratively though

Reply:
Not possible right now, as we're still running tests with Jest which require the `iframe-test` build. I'm all for removing it at some point though!